playbooks/ansible-cloud/k8s/downstream_sanity.yaml

```diff
@@ -0,0 +1,17 @@
+---
+- hosts: all
+  tasks:
+    - name: Ensure controller directory exists
+      file:
+        path: "{{ dest }}"
+        state: directory
+
+    - name: Clone redhat.openshift collection
+      git:
+        repo: "https://{{ ansible_collections_repo }}.git"
+        dest: "{{ dest }}"
```

Comment:
The repository will be at "~/{{ zuul.projects['github.com/ansible-collections/community.okd'].src_dir }}"